src/steps/updateLocalFiles.ts

```diff
@@ -8,7 +8,10 @@ interface ExistingPackageData {
 	version?: string;
 }
 
-export async function updateLocalFiles(options: Options) {
+export async function updateLocalFiles(
+	options: Options,
+	config = { resetPackageVersion: false },
```

Comment:
I've assumed we don't want to add another option for the user (extend `options`) and opted for a fully internal solution utilizing additional parameter).

Reply:
Yeah I think that makes sense. 👍